src/common/index.js

```diff
@@ -1,7 +0,0 @@
-// @flow
```

Comment:
I think this file should be kept, and its use more encouraged, since we do not use it a lot currently :slightly_smiling_face: